backend/src/nodes/nodes/external_stable_diffusion/img2img.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as ExternalStableDiffusionCategory
+from ...impl.external_stable_diffusion import (
+    decode_base64_image,
+    SamplerName,
+    STABLE_DIFFUSION_IMG2IMG_URL,
+    post,
+    encode_base64_image,
+)
+from ...node_base import NodeBase, group
+from ...node_factory import NodeFactory
+from ...properties.inputs import (
+    TextInput,
+    NumberInput,
+    SliderInput,
+    EnumInput,
+    ImageInput,
+)
+from ...properties.outputs import LargeImageOutput
+from typing import Optional
+
+
+@NodeFactory.register("chainner:external_stable_diffusion:img2img")
+class Img2Img(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Modify an image using Automatic1111"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Prompt", default="an astronaut riding a horse"),
+            TextInput("Negative Prompt").make_optional(),
+            SliderInput(
+                "Denoising Strength",
+                minimum=0,
+                default=0.75,
+                maximum=1,
+                slider_step=0.01,
+                controls_step=0.1,
+                precision=2,
+            ),
+            group("seed")(
+                NumberInput("Seed", minimum=0, default=42, maximum=4294967296)
+            ),
+            SliderInput("Steps", minimum=1, default=20, maximum=150),
+            EnumInput(SamplerName, default_value=SamplerName.EULER),
+            SliderInput(
+                "CFG Scale",
+                minimum=1,
+                default=7,
+                maximum=20,
+                controls_step=0.1,
+                precision=1,
+            ),
+            SliderInput("Width", minimum=64, default=512, maximum=2048),
+            SliderInput("Height", minimum=64, default=512, maximum=2048),
+            TextInput("Model Checkpoint Override").make_optional(),
+        ]
+        self.outputs = [
+            LargeImageOutput(),
```

Comment:
`LargeImageOutput` implicitly have side effects (the image is shown to the user). Please use a regular `ImageOutput`. Same for text to image.